specification/compute/resource-manager/Microsoft.Compute/DiagnosticRP/preview/2024-03-01-preview/diagnostic.json

```
@@ -0,0 +1,544 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "title": "ComputeDiagnosticResourceProviderClient",
+    "description": "The Compute Diagnostic Resource Provider Client",
+    "version": "2021-06-01-preview"
+  },
+  "host": "management.azure.com",
+  "schemes": [
+    "https"
+  ],
+  "consumes": [
+    "application/json"
+  ],
+  "produces": [
+    "application/json"
+  ],
+  "security": [
+    {
+      "azure_auth": [
+        "user_impersonation"
+      ]
+    }
+  ],
+  "securityDefinitions": {
+    "azure_auth": {
+      "type": "oauth2",
+      "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+      "flow": "implicit",
+      "description": "Azure Active Directory OAuth2 Flow",
+      "scopes": {
+        "user_impersonation": "impersonate your user account"
+      }
+    }
+  },
+  "paths": {
+    "/subscriptions/{subscriptionId}/providers/Microsoft.Compute/locations/{location}/diagnosticOperations/{operationId}": {
+      "get": {
+        "tags": [
+          "diagnosticOperations"
+        ],
+        "operationId": "DiagnosticOperations_Read",
+        "description": "Gets status of a Diagnostic operation issued for Disk Inspection.",
+        "parameters": [
+          {
+            "$ref": "#/parameters/SubscriptionIdParameter"
```

same as other places
```suggestion
            "$ref": "../../../../../../common-types/resource-management/v5/types.json#/parameters/SubscriptionIdParameter"
```

---

Updated